ts-nestjs-sequelize/test/pulumi/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as aws from "@pulumi/aws";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const stackPolicy: StackValidationPolicy = {
+    name: "redis-test",
+    description: "Redis integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const lambdas = args.resources.filter(r => r.isType(aws.lambda.Function));
+        if (lambdas.length !== 1) {
+            reportViolation(`Expected one lambda function but found ${lambdas.length}`);
+            return;
+        }
+
+        const lambda = lambdas[0].asType(aws.lambda.Function)!;
+        if (lambda.memorySize !== 512) {
+            reportViolation(
+                `Expected rds Instance '${lambda.name}' allocated storage to be '512' but found '${lambda.memorySize}'`);
```

Comment:
these messages say `rds Instance` but they're lambdas right

Reply:
yeah fixed this